main.js

```diff
@@ -0,0 +1,33 @@
+var categoryBoxWrapper = document.querySelector('.category-box-wrapper');
+var studyBox = document.querySelector('.study-box');
+var studyImage = document.querySelector('.study-image');
+var studyImageActive = document.querySelector('.study-image-active');
+var meditateBox = document.querySelector('.meditate-box');
+var meditateImage = document.querySelector('.meditate-image');
+var meditateImageActive = document.querySelector('.meditate-image-active');
+var exerciseBox = document.querySelector('.exercise-box');
+var exerciseImage = document.querySelector('.exercise-image');
+var exerciseImageActive = document.querySelector('.exercise-image-active');
+var minutesInput = document.querySelector('.minutes-input');
+var secondsInput = document.querySelector('.seconds-input');
+
+
+
+categoryBoxWrapper.addEventListener('click', activateCategory);
+
+// TODO: this function is ripe for refactoring but I'm not sure how yet
+function activateCategory(event) {
+  if (event.target.classList.contains('study-box')) {
+    studyImage.classList.toggle('hidden');
```

Comment:
Let's create a function for this hide/add and call here.

Reply:
I have the code for this; also need to modify based on the functionality

Reply:
cool